php/example_code/ses/Authorize_Sender.php

```diff
@@ -1,65 +1,67 @@
-<?php
-/**
- * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
- *
- * This file is licensed under the Apache License, Version 2.0 (the "License").
- * You may not use this file except in compliance with the License. A copy of
- * the License is located at
- *
- * http://aws.amazon.com/apache2.0/
- *
- * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
- * CONDITIONS OF ANY KIND, either express or implied. See the License for the
- * specific language governing permissions and limitations under the License.
- *
- *
- *
- *
- */
-
-require 'vendor/autoload.php';
-
-use Aws\SES\SESClient;
-use Aws\Exception\AwsException;
-
-//Create a SESClient
-$SesClient = new SesClient([
-    'profile' => 'default',
-    'version' => '2010-12-01',
-    'region'  => 'us-east-1'
-]);
-
-$identity = "arn:aws:ses:us-east-1:123456789012:identity/example.com";
-$other_aws_account = "0123456789";
-$policy = "{
- 'Id':'ExampleAuthorizationPolicy',
- 'Version':'2012-10-17',
- 'Statement':[
-    {
-     'Sid':'AuthorizeAccount',
-     'Effect':'Allow',
-     'Resource':'" . $identity . "',
-     'Principal':{
-       'AWS':['" . $other_aws_account . "' ]
-      },
-     'Action':[
-       'SES:SendEmail',
-       'SES:SendRawEmail'
-      ]
-    }
-  ]
-}";
-$name = "policyName";
-
-try {
-    $result = $SesClient->putIdentityPolicy([
-        'Identity' => $identity,
-        'Policy' => $policy,
-        'PolicyName' => $name,
-]);
-    var_dump($result);
-} catch (AwsException $e) {
-    // output error message if fails
-    echo $e->getMessage();
-    echo "\n";
-}
+<?php
+/**
+ * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * This file is licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ *
+ *
+ *
+ */
+
+require 'vendor/autoload.php';
+
+use Aws\SES\SESClient;
+use Aws\Exception\AwsException;
+
+//Create a SESClient
+$SesClient = new SesClient([
+    'profile' => 'default',
+    'version' => '2010-12-01',
+    'region' => 'us-east-1'
+]);
+
+$identity = "arn:aws:ses:us-east-1:123456789012:identity/example.com";
+$other_aws_account = "0123456789";
+$policy = <<<EOT
+{
+  "Id":"ExampleAuthorizationPolicy",
+  "Version":"2012-10-17",
+  "Statement":[
+    {
+      "Sid":"AuthorizeAccount",
+      "Effect":"Allow",
+      "Resource":"" . $identity . "",

```

Comment:
Heredoc handles variable substitution, this line should be `"Resource":"$identity",` and others should be updated accordingly.